blobstore/config.go

```diff
@@ -0,0 +1,5 @@
+package blobstore
+
+// these are the default values of limits if not predefined by user from env
+const DEFAULT_ZIP_DOWNLOAD_SIZE_LIMIT = 5     //gb
```

Comment:
These should follow the normal naming convention for variables, even though they are global, i.e., defaultZipDownloadSizeLimit

Reply:
I was going with the C conventions of config variables being in MACRO_CASE, I thought go was the same!

Reply:
Not to my knowledge